weave/scorers/llm_utils.py

```diff
@@ -83,7 +83,7 @@ def create(
 
 
 def embed(
-    client: _LLM_CLIENTS, model_id: str, texts: Union[str, list[str]], **kwargs: Any
+    client: _LLM_CLIENTS, model_id: str, texts: str | list[str], **kwargs: Any
```

Comment:
seems like this replacement could be everywhere - why just this one spot?

Reply:
It's because these annotations are opt-in until py310 (we still support py39)

It only kicks in for places where we explicitly do `from __future__ import annotations`